packages/katello/katello/backup.rb

```diff
@@ -343,6 +343,7 @@ def generate_metadata
       File.open('metadata.yml', 'w') do |metadata_file|
         metadata_file.puts system_facts.to_yaml
```

Comment:
@cfouant I was more thinking of doing something like this - here, we could add incremental as a key in the metadata yaml file:

```
system_facts = {os_version: os_version, plugin_list: plugins, rpms: rpms, incremental: <boolean_for_incremental>}
```

Which means now a backup can be determined as incremental based on its `metadata.yaml`

Then in restore, get rid of the `-i` option and parse the `metadata.yaml` file for the `incremental` boolean and use that to determine if we need to run `reset_katello`